linode/manager
# Fix: [M3-6923] - Regressions with sidebar menu

## Description 📝
We had several regressions with the sidebar menu stemming from adding the AGLB logo, and in general the behavior of the akamai logo:

This PR fixes
- Active menu item icon color
- Akamai logo spacing => desktop, collapsed > hover
- Akamai logo text => animation was bad and janky
- Menu items jumpy (esp object storage) => desktop, collapsed > hover

The menu now looks more stable and behaves consistently
See self review for more context

## How to test 🧪
Test all states of Sidebar Menu
- desktop, open
- desktop collapsed
- desktop collapsed > hover
- mobile, collapsed
- mobile, open

Confirm improvement over current development branch AND improvement over production behavior
